components/client/src/test/unit/java/com/hotels/styx/client/loadbalancing/strategies/BusyConnectionsStrategyTest.java

```diff
@@ -67,46 +70,46 @@ public double oneMinuteRateForStatusCode5xx(Origin origin) {
     public void favoursOriginsWithLessBusyConnectionCount() {
         ConnectionPool.Settings settings = ConnectionPoolSettings.defaultConnectionPoolSettings();
 
-        ConnectionPool poolOne = new StubConnectionPool(ORIGIN_ONE, settings).withBusyConnections(4);
-        ConnectionPool poolTwo = new StubConnectionPool(ORIGIN_TWO, settings).withBusyConnections(3);
-        ConnectionPool poolThree = new StubConnectionPool(ORIGIN_THREE, settings).withBusyConnections(6);
+        RemoteHost poolOne = new RemoteHostWrapper(ORIGIN_ONE.id(), ORIGIN_ONE, new StubConnectionPool(ORIGIN_ONE, settings).withBusyConnections(4), mock(StyxHostHttpClient.class));
```

Comment:
these calls to `new RemoteHostWrapper` are creating very long lines. Perhaps a convenience method can be extracted?